pkg/products/observability/reconciler.go

```diff
@@ -19,13 +28,33 @@ import (
 	"github.com/integr8ly/integreatly-operator/pkg/resources/events"
 	"github.com/integr8ly/integreatly-operator/pkg/resources/marketplace"
 	"github.com/integr8ly/integreatly-operator/version"
+	routev1 "github.com/openshift/api/route/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/tools/record"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	defaultInstallationNamespace = "observability"
+
+	// alert manager configuration
+	alertManagerRouteName                   = "kafka-alertmanager"
```

Comment:
what do you think about renaming this one? In AMO seems we use `alertmanager-route` my preference would be just `alertmanager` because we know it's a route object.

Reply:
:+1:

Reply:
This is hard coded in OO https://github.com/bf2fc6cc711aee1a0c2a/observability-operator/blob/main/controllers/model/alertmanager_resources.go#L34, so can't changed it here